### PR TITLE
Spanish v2.0 rollout [Closes #2489]

### LINKED ISSUE #2489: Site Translation: Update Spanish to v2.0
**Is your feature request related to a problem? Please describe.**

We have our first v2.0 translation complete!
https://crowdin.com/project/ethereumfoundation/es-EM#

**Describe the solution you'd like**

Download the files from Crowdin for v2.0 & add them to the site.

**Additional context**

Here's a similar issue for reference: https://github.com/ethereum/ethereum-org-website/issues/2070
NOTE: since this is a new version (v2.0) & our codebase has changed the process will be slightly different here.

### PR DESCRIPTION
**Pull request by wackerow**
## Description
This PR rolls out the Spanish translations provided from Crowdin to upgrade content to v2.0. 
- /assets
- /dapps
- /developers/docs
- /developers/learning-tools
- /developers/local-environment
- /developers/tutorials
- /eth
- /eth2
- /eth2/deposit-contract
- /eth2/get-involved
- /eth2/get-involved/bug-bounty
- /eth2/staking
- /eth2/vision
- /get-eth
- /languages
- /stablecoins
- /wallets
- /wallets/find-wallet
- /what-is-ethereum

## Notes:
- `/dapps`, `developers/local-environment`, and `developers/learning-tools`: The cards on these pages have buttons and pills that are not translated
- `/get-eth`: "Type where you live..." not translated
- Wallet listings continually update, and new ones do not have description translated
- (Launchpad links do not carry over language yet, will need address this when Launchpad translations branch merges)

## Related Issue #2489

### CODE REVIEW

**Comment by wackerow** [comment]
### Netlify Preview (direct to Spanish homepage)
https://deploy-preview-2496--ethereumorg.netlify.app/es/

@samajammin @ryancreatescopy @sorumfactory Take a look, let me know if you see any issues =)

File: gatsby-node.js [hunk 1]
```diff
@@ -230,6 +230,37 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     })
   })
 
+  // Create contentVersion v2.0 pages
```
Comment (samajammin) [inline gatsby-node.js:233]: This should replace the block to create `englishOnlyPages`

File: gatsby-node.js [hunk 2]
```diff
@@ -230,6 +230,37 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
...
+  const contentV2Pages = [`eth`, `dapps`, `wallets/index`, `what-is-ethereum`]
+  const contentV2Languages = supportedLanguages.filter(
+    (lang) => getLangContentVersion(lang) === 2.0
```
Comment (samajammin) [inline gatsby-node.js:236]:
```suggestion
    (lang) => getLangContentVersion(lang) >= 2.0
```
Update to GTE (so it includes English, which is 2.4)

**Comment by samajammin** [comment]
@wackerow this looks great for the most part.

After looking through some of these pages though I think it's worth investing time into fixing our TranslationBanner logic. e.g. currently on all updated pages, the banner says "You're viewing this page in English..." even though the page is clearly in Spanish, e.g. https://deploy-preview-2496--ethereumorg.netlify.app/es/what-is-ethereum/

Image 2021-03-01 at 4 35 55 PM

I think this could be a confusing UX.

**Review by samajammin — APPROVED** [review]

**Pull request merged**